nuit/utils.py

```diff
@@ -13,10 +13,24 @@ def _closure(fn):
         return fn.__closure__
 
 
+def _code(fn):
+    '''
+    Python 2 and Python 3 compatibility function
+    '''
+    try:
+        return fn.func_code
+    except AttributeError:
+        return fn.__code__
+
+
 def has_closure(fn):
     return hasattr(fn, 'func_closure') or hasattr(fn, '__closure__')
 
 
+def has_code(fn):
```

Comment:
Does function code tag checking was actually trying to check if it is a function? I didn't looked at the code logic that deep, just done shallow refactoring to make it work